docs/src/main/sphinx/connector/memory.rst

```diff
@@ -78,7 +63,24 @@ DROP TABLE
 
 Upon execution of a ``DROP TABLE`` operation, memory is not released
 immediately. It is instead released after the next write operation to the
-connector.
+catalog.
+
+.. _memory_dynamic_filtering:
+
+Dynamic filtering
```

Comment:
Suggestion: Would you consider putting this under a "Performance" section to be in-line with other connectors? Even though it's just the one sub-section.

Reply:
I would .. but in the past that was vetoed.. so I am leaving this out of this PR..